### ISSUE TITLE
Suddenly running into 403

### ISSUE THREAD

**Issue by digitalkram** [darekkay/todoist-export#22]
hi @darekkay,

thanks for this incredibly useful tool! I configured it a couple of days ago and it was working.

Since yesterday (with no changes to the todoist app or the local config) it suddenly stopped working:

```
Request failed with status code 403

Error: Request failed with status code 403
    at createError (/Users/myusername/todoist-migration/todoist-export/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/Users/myusername/todoist-migration/todoist-export/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/Users/myusername/todoist-migration/todoist-export/node_modules/axios/lib/adapters/http.js:236:11)
    at IncomingMessage.emit (node:events:525:35)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

Any idea what might be wrong? Can I somehow provide more diagnostic data?

Kind regards
digitalkram

**Comment by darekkay** [comment]
This is probably a duplicate of #20, which has been fixed in #21 and merged today.

**Comment by digitalkram** [comment]
Thanks a lot @darekkay! Confirmed - #21 is the fix.
